content/deploy/maintain/audit.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Maintain Audit PostgreSQL'
+date: '2024-03-26T11:20:56-03:00'
+categories: ["how-to"]
+description: How to maintain Audit PostgreSQL on your Rhize deployment
+weight: 300
+menu:
+  main:
+    parent: maintain
+    identifier:
+---
+
+This guide shows you the procedure to maintain your Audit Postgres database on your Rhize Kubernetes deployment.
```

Comment:
I think we can be more specific than "Maintain": what is the actual action being documented? That's what we should title it.